diff --git a/extensions/desktop/gnome/launcher-specific b/extensions/desktop/gnome/launcher-specific
index ff0b5453..1f0c5fd8 100755
--- a/extensions/desktop/gnome/launcher-specific
+++ b/extensions/desktop/gnome/launcher-specific
@@ -10,7 +10,6 @@ if [ "$wayland_available" = true ]; then
   # Does not hurt to specify this as well, just in case
   export QT_QPA_PLATFORM=wayland-egl
 fi
-append_dir GTK_PATH "$SNAP_DESKTOP_RUNTIME/usr/lib/$ARCH/gtk-3.0"
 
 # ibus and fcitx integration
 GTK_IM_MODULE_DIR="$XDG_CACHE_HOME/immodules"
