--- a/configure.ac	2022-02-22 22:13:08.000000000 +0000
+++ b/configure.ac	2022-07-26 15:02:43.681462618 +0000
@@ -64,7 +64,7 @@
 m4_define([gegl_required_version],
           [gegl_major_minor_version.gegl_micro_version])
 m4_define([gexiv2_required_version], [0.12.2])
-m4_define([glib_required_version], [2.68.0])
+m4_define([glib_required_version], [2.56.0])
 m4_define([gtk_mac_integration_required_version], [2.0.0])
 m4_define([gtk_required_version], [3.22.29])
 m4_define([gtkdoc_required_version], [1.0])