--- a/tools/g-ir-tool-template.in	2018-04-03 00:54:43.914958850 +0100
+++ b/tools/g-ir-tool-template.in	2018-04-03 00:53:30.773509035 +0100
@@ -62,5 +62,12 @@
 
 sys.path.insert(0, pylibdir)
 
+stagedir = os.getenv('SNAPCRAFT_STAGE', None)
+if stagedir is not None:
+    sys.path.insert(0, os.path.join(stagedir, 'usr', 'lib', 'gobject-introspection'))
+    stagedatadir = os.path.join(stagedir, 'usr', 'share')
+    if os.path.isdir(os.path.join(stagedatadir, 'gir-1.0')):
+        builtins.__dict__['DATADIR'] = stagedatadir
+
 from giscanner.@TOOL_MODULE@ import @TOOL_FUNCTION@
 sys.exit(@TOOL_FUNCTION@(sys.argv))
