diff --git a/gtk/launcher-specific b/gtk/launcher-specific
index 026665e..ce5c625 100644
--- a/gtk/launcher-specific
+++ b/gtk/launcher-specific
@@ -5,12 +5,6 @@
 # select gtk version
 . $SNAP/flavor-select
 
-if [ "$USE_gtk3" = true ]; then
-  export GTK_PATH=$RUNTIME/usr/lib/$ARCH/gtk-3.0
-else
-  export GTK_PATH=$RUNTIME/usr/lib/$ARCH/gtk-2.0
-fi
-
 # ibus and fcitx integration
 GTK_IM_MODULE_DIR=$XDG_CACHE_HOME/immodules
 export GTK_IM_MODULE_FILE=$GTK_IM_MODULE_DIR/immodules.cache
